dev/page-building-studio/schemaTypes/index.ts

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I'm unsure if it's OK to suppress this error. I only added `react-icons` to this workspace. Isn't that OK?